### ISSUE TITLE
Difference between aval.Syn_degree() in alpha0.5 vs readme

### ISSUE THREAD

**Issue by slarson** [openworm/PyOpenWorm#101]
Output we claim in the readme:

```
#show how many chemical synapses go in and out of AVAL
>>> aval.Syn_degree()
74
```

Output when run on my system on command line:
```
>>> aval.Syn_degree()
91
```

I think 74 is the right answer... we need to check to confirm to be sure.

**Comment by travs** [comment]
Looking into this right now, and dumping info for later use.
The file NeuronConnect.xls is where we get the data for connections. Getting the information for AVAL, we have [this](http://hastebin.com/oyexevupox.lua).

With respect to that data, and using the data legend above the diagrams [here](http://www.wormatlas.org/neuronalwiring.html#Connectivitydata), `S+Sp+R+Rp` will give the total number of chemical connection pairings (i.e. chemical connections between two distinct neurons) and, for each pair, `Nbr` is the number of synapses between the two neurons.

Summing the AVAL `Nbr` for `S+Sp+R+Rp` we get 382.

If we ignore `Nbr` and just want the synapse pairings, we still get 114.

**Comment by travs** [comment]
OK, the above difference (107 vs 114) for AVAL is caused by the fact that `insert_worm.py` is creating only one Connection for a few of the synapse pairs.

This occurs when something like this occurs in the database:
```
AVAL   SDQL   R    1
AVAL   SDQL   Rp  1
```

Finding a solution now.

**Comment by slarson** [comment]
Closed by #115
